docs/getting-started-guides/clc.md

```diff
@@ -0,0 +1,317 @@
+# Kubernetes on CenturyLink Cloud
```

Comment:
Remove the "# title" from here and put it in guides.yml; that's where we define page titles; keeps the left nav and the page titles in sync